lib/shoryuken/manager.rb

```diff
@@ -3,7 +3,7 @@ class Manager
     include Util
 
     BATCH_LIMIT        = 10
-    HEARTBEAT_INTERVAL = 0.1
+    HEARTBEAT_INTERVAL = 1
```

Comment:
Something like this:

```ruby
def dispatch_now
  while true
    if ready.zero?
      return unless (queue = @polling_strategy.next_queue)
  
      logger.debug { "Ready: #{ready}, Busy: #{busy}, Active Queues: #{@polling_strategy.active_queues}" }

      batched_queue?(queue) ? dispatch_batch(queue) : dispatch_single_messages(queue)
    else
      sleep MINIMUM_WAIT # 0.05 or something to prevent CPU pegging
    end
  end
end
```